telegram/messageentity.py

```diff
@@ -32,6 +32,8 @@ class MessageEntity(TelegramObject):
         length (:obj:`int`): Length of the entity in UTF-16 code units.
         url (:obj:`str`): Optional. Url that will be opened after user taps on the text.
         user (:class:`telegram.User`): Optional. The mentioned user.
+        language (:obj:`str`): Optional. For “pre” only, the programming language of the entity
```

Comment:
Can you use "\`\`pre\`\`" instead of “pre”? Noticed b/c the py2.7 tests complain about a non-asci character, but it also won't render correctly in rtd. Some below. Didn't check the rest, just noticed from the tests …

Reply:
done